Jenkinsfile.cd

```diff
@@ -1251,6 +1258,7 @@ def downloadPackagingUtils() {
 
 def setupRust() {
     sh "rustup default 1.29.0"
+    sh "rustup component add clippy-preview"
```

Comment:
As for now `setupRust` isn't used in ubuntu/redhat branches. As result future `cargo clippy` will fail. Please add `setupRust` for linuxTesting and similar steps

Reply:
I decided it would be better to run static analysis as a separate stage. Adding clippy is abstracted to a different section now and should work on linux systems.